baremaps-server/src/main/java/org/apache/baremaps/server/GeocoderResource.java

```diff
@@ -54,15 +54,16 @@ public GeocoderResource(SearcherManager searcherManager) {
 
   @GET
   @javax.ws.rs.Path("/api/geocoder")
-  public Response getIpToLocation(
+  public Response searchLocations(
       @QueryParam("queryText") String queryText,
       @QueryParam("countryCode") @DefaultValue("") String countryCode,
       @QueryParam("limit") @DefaultValue("10") int limit) throws IOException {
     if (queryText == null) {
       throw new WebApplicationException(Response.status(Response.Status.BAD_REQUEST)
           .entity("The queryText parameter is mandatory").build());
     }
-    var query = new GeonamesQueryBuilder().queryText(queryText).countryCode(countryCode).build();
+    var query = new GeonamesQueryBuilder()
+        .queryText(queryText).countryCode(countryCode).withScoringByPopulation().build();
```

Comment:
Should we use the key word "with" before each method? e.g. withQueryText.

Reply:
dropped the usage of prefix "with" to match builder naming based on field names. 5caf68cbe7d30f290bf2e61568bb7876758a347a